commune_app/migrations/0001_initial.py

```diff
@@ -116,10 +117,18 @@ class Migration(migrations.Migration):
                     auto_created=True,
                     primary_key=True,
                     serialize=False,
-                    verbose_name='ID')),
-                ('name', models.TextField(max_length=150, unique=True)),
+                    verbose_name='ID'
+                    )),
+                ('name', models.TextField(
+                    max_length=100,
+                    unique=True,
+                    validators=[commune_app.all_models.communes.validate_name]
+                    )),
```

Comment:
you don't need the validator you can use the built-in validator when you use CharField
https://stackoverflow.com/questions/7354588/whats-the-difference-between-charfield-and-textfield-in-django
```suggestion
                ('name', models.CharField(
                    max_length=100,
                    unique=True
                    )),
```